source/docs/tools/draw-tool.md

```diff
@@ -2,52 +2,96 @@
 
 # Draw Tool
 
-Molecules are built and edited with the Draw Tool, which allows "free-hand" sketching of new molecules.
+![The icon of the Draw Tool in light mode.](../../_static/icon_draw.svg)
 
-```{tip} **New in 2.0**
```

Comment:
I think this would still be useful in the draw tool documentation, since many people go to it first (e.g., "draw" is not the only way to build up molecules in 2.0)

Reply:
I don't quite follow what you mean here? If you mean the callout encouraging people to use the Template Tool, it's on lines 11-13